.idea/codeStyles/Project.xml

```diff
@@ -1,5 +1,23 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <JetCodeStyleSettings>
+      <option name="PACKAGES_TO_USE_STAR_IMPORTS">
+        <value>
+          <package name="java.util" alias="false" withSubpackages="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
+          <package name="io.ktor" alias="false" withSubpackages="true" />
+        </value>
+      </option>
+      <option name="PACKAGES_IMPORT_LAYOUT">
+        <value>
+          <package name="" alias="false" withSubpackages="true" />
+          <package name="java" alias="false" withSubpackages="true" />
+          <package name="javax" alias="false" withSubpackages="true" />
+          <package name="kotlin" alias="false" withSubpackages="true" />
+          <package name="" alias="true" withSubpackages="true" />
+        </value>
+      </option>
+    </JetCodeStyleSettings>
```

Comment:
What is this section that you added here? Was this the thing that you said kept getting added, but you weren't sure why?

Reply:
Yes, it looks like I accidently pushed it.

Reply:
Do you want to remove it?

Reply:
How do I remove it?

Reply:
I guess you can go into the file and delete those lines.

Reply:
Is it fine to mess with a config file?

Reply:
Good point. Maybe you should just leave it as is and check it in.